src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.form {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.label::before {
+  content: "";
+  position: absolute;
+  background-image: url("images/Search.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
```

Comment:
Can be combined with [`background` shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/background)